app/views/katello/api/v2/subscription_facet/subscription_status.json.rabl

```diff
@@ -0,0 +1,3 @@
+
+attributes :to_global => :color_code
```

Comment:
I got those backwards, however, I will say these still are confusing API wise:

```
"subscription_status": {
          "color_code": 0,
          "label": "Fully entitled"
        }
```

Label I can somewhat interpret though still strange to me, and color_code? How do I interpret or reason about that?